readme.md

```diff
@@ -22,25 +25,94 @@ of the code between the tags for that feature.  Following that, you should remov
 `# START_FEATURE` and `# END_FEATURE` comments from the codebase.
 Following that, you should run `pip-compile`.
 
-The follwing is a list of tagged features in this repo:
+The following is a list of tagged features in this repo:
 
 ```
 bootstrap_messages
 django_social
 crispy_forms
+django_react
+debug_toolbar
+sentry
 ```
 
 The codebase also has a number of reference examples.  These are all marked with the comment:
 ```
 # TODO: delete me; this is just a reference example
 ```
 
+
+# Feature Descriptions
+
+
+## Django-React
+
+
+### Additional Setup
+
+
+When using this feature, make sure to install the Node.js requirements using the manager of your choice 
+(either `npm install` or `yarn install` will work) before proceeding with development.
+
+
+### Special Consideration for Running
+
+
+For development on localhost when using Django-React, you should run the following command in a separate terminal to
+your standard `runserver` command.
+    
+- `nwb serve --no-vendor`
+
+If you have configured everything correctly, you should see each command complete and notify you
+that the project is ready to be viewed.
+
+
+### Adding a new React component
+
+
+In this paradigm, React components are compiled and injected into the standard Django template. This means we can take 
+advantage of the built-in templating functionality of Django and, with a bit of elbow grease, use the power of React to
+make those templates responsive.
+
+`django-react-loader` uses the same basic pattern for any component:
+
+1. First, ensure that the library is loaded in your template: `{% load django_react_components %}`
+2. Next, ensure that you have rendered the React runtime bundle: `{% render_bundle 'runtime' %}`
+   - Note that you only have to do this once per page where React components will be used. 
+3. Finally, load your React component on the page. `{% react_component 'Home' id='home' %}`
+    - You can add any number of props as named keywords, e.g. `{% react_component 'Home' id='home' prop1=value_from_context %}`
+
+
+### Preparing for deployment
+
+
+To prepare the application for deployment, run `nwb build --no-vendor`. This will generate a `webpack_bundles` folder
+in your `/static` folder populated with the compiled React components. You may now deploy the app as you would any standard
+Django application. 
```

Comment:
I'd actually recommend adding the compilation step to the deploy configuration instead of building it on a local machine.

But since we don't yet have that in the sample-django-app repo, maybe just mention that that's the preferred option?